### PR TITLE
Mailchimp API with Google Apps Script

### PR DESCRIPTION
**Pull request by labnol**
I have included support for Mailchimp in my [Mail Merge with Attachments](https://gsuite.google.com/marketplace/app/mail_merge_with_attachments/223404411203) add-on for Gmail using the OAuth2 Library for Google Apps Script.

To get started, create your own set of credentials from your Mailchimp account as described [here](https://mailchimp.com/developer/guides/how-to-use-oauth2/#Step_1%3A_Register_your_application).

- In your Mailchimp account, navigate to the Account page.
- In the drop-down menu, select Extras, and then Registered Apps.
- Click Register an App.
- In the appropriate fields, enter your application information and click Create.

Screenshot 2020-02-23 at 2 07 49 PM

### CODE REVIEW

File: samples/Mailchimp.gs [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/*
+ * This sample demonstrates how to configure the library for the Mailchimp API.
+ * Instructions on how to generate OAuth credentuals is available here:
+ * https://mailchimp.com/developer/guides/how-to-use-oauth2/#Step_1%3A_Register_your_application
+ */
+
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+var OAUTH_HOST = 'login.mailchimp.com';
```
Comment (erickoledadevrel) [inline samples/Mailchimp.gs]: After scanning the docs, it seems like the OAuth host is always this value. If there is no need for it to change then I'd prefer the variable be removed and the value just be baked into the URLs below.

File: samples/Mailchimp.gs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ * Authorizes and makes a request to the Docusign API.
+ */
+function run() {
+  var service = getService();
+  if (service.hasAccess()) {
+    // Retrieve the account ID and base URI from storage.
+    var storage = service.getStorage();
+    var dc = storage.getValue('dc');
+
+    // Make a request to retrieve the account information.
```
Comment (erickoledadevrel) [inline samples/Mailchimp.gs]: "account" => "campaign"

Comment (labnol) [reply samples/Mailchimp.gs]: Good catch

File: samples/Mailchimp.gs (hunk 1 above)
Comment (labnol) [reply samples/Mailchimp.gs]: That’s a good suggestion.

**Pull request merged**